### PR TITLE
Release notes for 2.18.0

### PR DESCRIPTION
**Pull request by abrookins**
### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] If this pull request adds new functionality, it includes unit tests that cover the changes
- [ ] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed.

For new functions or classes in the Python SDK:

- [ ] This pull request includes helpful docstrings.
- [ ] If a new Python file was added, this pull request contains a stub page in the Python SDK docs and an entry in `mkdocs.yml` navigation.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,93 @@
...
+- Add `/events/filter` and `/events/count-by` route trees — https://github.com/PrefectHQ/prefect/pull/12736
+- Allow for creating automations via deployments when experimental events is on — https://github.com/PrefectHQ/prefect/pull/12701
+- Add ability to stream out events via websocket — https://github.com/PrefectHQ/prefect/pull/12744
+- Implementing the `pause-automation` and `resume-automation` actions — https://github.com/PrefectHQ/prefect/pull/12738
+- Add an Automations CLI — https://github.com/PrefectHQ/prefect/pull/12754
+- Rename `prefect-cloud.*` events and labels to `prefect.*` — https://github.com/PrefectHQ/prefect/pull/12755
+- Add ability to emit events to an ephemeral Prefect server — https://github.com/PrefectHQ/prefect/pull/12762
+- Disable `events` and `automations` API routes when experimental events setting is not enabled — https://github.com/PrefectHQ/prefect/pull/12777
+- Add compatibility tests for client and server triggers and actions — https://github.com/PrefectHQ/prefect/pull/12778
+- Disable the automations integration flows for Prefect Cloud — https://github.com/PrefectHQ/prefect/pull/12784
```
Comment (desertaxle) [inline RELEASE-NOTES.md:40]: In the past couple of releases we've been further dividing these PRs into events and pydantic for people who are interested in in progress features.

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,96 @@
...
+- Simplify tutorial — https://github.com/PrefectHQ/prefect/pull/12765
+- Remove disclaimer for Python 3.12 experimental support — https://github.com/PrefectHQ/prefect/pull/12771
+- Clarify deployment trigger examples — https://github.com/PrefectHQ/prefect/pull/12782
+- Remove Prefect-managed integration libraries to be archived from the integrations catalog — https://github.com/PrefectHQ/prefect/pull/12781
+- Fix broken link to Push Work Pool Guide — https://github.com/PrefectHQ/prefect/pull/12748
+- Fix minor restructure to improve legibility of `work pools` tutorial — https://github.com/PrefectHQ/prefect/pull/12747
+- Fix `typing` import and typos in Tasks tutorial — https://github.com/PrefectHQ/prefect/pull/12746
+- Simplify installation — https://github.com/PrefectHQ/prefect/pull/12772
+
+### Uncategorized
```
Comment (serinamarie) [inline RELEASE-NOTES.md:75]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,96 @@
...
+* @takashimakazuki made their first contribution in https://github.com/PrefectHQ/prefect/pull/12761
+  
+### Integrations
+- Add support for a capacity provider — https://github.com/PrefectHQ/prefect-aws/pull/407
+- Improve error handling for task creation — https://github.com/PrefectHQ/prefect-aws/pull/406
+
+### Contributors
+- @avriiil
+- @hboehmer-IW
+- @takashimakazuki
```
Comment (serinamarie) [inline RELEASE-NOTES.md:89]: these are dupes since they're all new contributors so we can remove this
```suggestion
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,96 @@
...
+
+**All changes**: https://github.com/PrefectHQ/prefect/compare/2.17.1...2.18.0
+
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:93]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,96 @@
...
+## Release 2.18.0
+
+### Breaking Changes
+- Remove deprecated ability to run `prefect deploy` with `deployment.yaml` file — https://github.com/PrefectHQ/prefect/pull/12731
+- Remove deprecated ability to pass `-f/--flow` as option to `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/12732
+- Remove deprecated `projects` from `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/12737
+- Remove deprecated `--ci` option from `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/12740
+
+### Enhancements
+- Introduce self-validating types — https://github.com/PrefectHQ/prefect/pull/12707
```
Comment (serinamarie) [inline RELEASE-NOTES.md:12]: this can go in Pydantic section

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,96 @@
...
+- Allow returning to account selection in `prefect cloud login` and `workspace set` — https://github.com/PrefectHQ/prefect/pull/12717
```
Comment (serinamarie) [inline RELEASE-NOTES.md:13]:
```suggestion
- Improve account selection in `prefect cloud login` and `workspace set` — https://github.com/PrefectHQ/prefect/pull/12717
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,96 @@
...
+- Add default posture to `EventTrigger` schema — https://github.com/PrefectHQ/prefect/pull/12764
```
Comment (serinamarie) [inline RELEASE-NOTES.md:14]: probably can go in events section

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,96 @@
...
+
+### Fixes
+- Better flow validation error — https://github.com/PrefectHQ/prefect/pull/12715
```
Comment (serinamarie) [inline RELEASE-NOTES.md:17]:
```suggestion
- Raise clearer flow validation error — https://github.com/PrefectHQ/prefect/pull/12715
```

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,96 @@
...
+- Fix import syntax in `variables.Variable` example — https://github.com/PrefectHQ/prefect/pull/12727
+- Fix typo in How-to Guide document — https://github.com/PrefectHQ/prefect/pull/12761
+- Fix writing events for SQLite + SQLAlchemy<2 — https://github.com/PrefectHQ/prefect/pull/12679
+- Fix a flake with the SQLite version check test — https://github.com/PrefectHQ/prefect/pull/12690
+- Fix a test flake in flow run lateness — https://github.com/PrefectHQ/prefect/pull/12698
+- Remove `version` from `docker-compose.yml`, as it is now deprecated — https://github.com/PrefectHQ/prefect/pull/12706
+- Fix types NonNegativeDuration / PositiveDuration — https://github.com/PrefectHQ/prefect/pull/12711
```
Comment (serinamarie) [inline RELEASE-NOTES.md:24]: probably pydantic?

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,96 @@
...
+### Enhancements
+- Introduce self-validating types — https://github.com/PrefectHQ/prefect/pull/12707
+- Allow returning to account selection in `prefect cloud login` and `workspace set` — https://github.com/PrefectHQ/prefect/pull/12717
+- Add default posture to `EventTrigger` schema — https://github.com/PrefectHQ/prefect/pull/12764
+
+### Fixes
+- Better flow validation error — https://github.com/PrefectHQ/prefect/pull/12715
+- Fix import syntax in `variables.Variable` example — https://github.com/PrefectHQ/prefect/pull/12727
+- Fix typo in How-to Guide document — https://github.com/PrefectHQ/prefect/pull/12761
+- Fix writing events for SQLite + SQLAlchemy<2 — https://github.com/PrefectHQ/prefect/pull/12679
```
Comment (serinamarie) [inline RELEASE-NOTES.md:20]: events section?

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,96 @@
...
+- Fix a flake with the SQLite version check test — https://github.com/PrefectHQ/prefect/pull/12690
+- Fix a test flake in flow run lateness — https://github.com/PrefectHQ/prefect/pull/12698
+- Remove `version` from `docker-compose.yml`, as it is now deprecated — https://github.com/PrefectHQ/prefect/pull/12706
```
Comment (serinamarie) [inline RELEASE-NOTES.md:23]: may be maintenance and therefore not needed?

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -1,5 +1,96 @@
...
+- Fix types NonNegativeDuration / PositiveDuration — https://github.com/PrefectHQ/prefect/pull/12711
+- Exclude job_variables when exclude=None — https://github.com/PrefectHQ/prefect/pull/12712
+- Remove all infra overrides-related flags — https://github.com/PrefectHQ/prefect/pull/12742
```
Comment (serinamarie) [inline RELEASE-NOTES.md:26]:
```suggestion
- Remove experimental flags on infrastructure overrides — https://github.com/PrefectHQ/prefect/pull/12742
```

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -1,5 +1,96 @@
...
+- Make `TriggeredAction.firing` required — https://github.com/PrefectHQ/prefect/pull/12697
+- Add an Actions service — https://github.com/PrefectHQ/prefect/pull/12699
+- Implement the `call-webhook` action and adds all Action client-side schemata — https://github.com/PrefectHQ/prefect/pull/12728
+- Implement `change-flow-run-state`, `cancel-flow-run`, and `suspend-flow-run` — https://github.com/PrefectHQ/prefect/pull/12730
+- Add functions for querying and counting events — https://github.com/PrefectHQ/prefect/pull/12696
+- Implement the `pause-deployment` and `resume-deployment` actions — https://github.com/PrefectHQ/prefect/pull/12733
+- Add `/events/filter` and `/events/count-by` route trees — https://github.com/PrefectHQ/prefect/pull/12736
+- Allow for creating automations via deployments when experimental events is on — https://github.com/PrefectHQ/prefect/pull/12701
+- Add ability to stream out events via websocket — https://github.com/PrefectHQ/prefect/pull/12744
+- Implementing the `pause-automation` and `resume-automation` actions — https://github.com/PrefectHQ/prefect/pull/12738
```
Comment (serinamarie) [inline RELEASE-NOTES.md:46]:
```suggestion
- Implement the `pause-automation` and `resume-automation` actions — https://github.com/PrefectHQ/prefect/pull/12738
```

File: RELEASE-NOTES.md [hunk 14]
```diff
@@ -1,5 +1,96 @@
...
+- Add an Automations CLI — https://github.com/PrefectHQ/prefect/pull/12754
```
Comment (serinamarie) [inline RELEASE-NOTES.md:47]:
```suggestion
- Add automations CLI — https://github.com/PrefectHQ/prefect/pull/12754
```

File: RELEASE-NOTES.md (hunk 11 above)
Comment (serinamarie) [inline RELEASE-NOTES.md:23]: maintenance fixes probably don't need to be in the release

File: RELEASE-NOTES.md [hunk 15]
```diff
@@ -1,5 +1,96 @@
...
+- Simplify-api-connection — https://github.com/PrefectHQ/prefect/pull/12725
+- Add `.serve`, `.deploy`, and composite trigger examples to deployment triggers docs — https://github.com/PrefectHQ/prefect/pull/12743
+- Update automations images — https://github.com/PrefectHQ/prefect/pull/12752
+- Simplify tutorial — https://github.com/PrefectHQ/prefect/pull/12765
+- Remove disclaimer for Python 3.12 experimental support — https://github.com/PrefectHQ/prefect/pull/12771
+- Clarify deployment trigger examples — https://github.com/PrefectHQ/prefect/pull/12782
+- Remove Prefect-managed integration libraries to be archived from the integrations catalog — https://github.com/PrefectHQ/prefect/pull/12781
+- Fix broken link to Push Work Pool Guide — https://github.com/PrefectHQ/prefect/pull/12748
+- Fix minor restructure to improve legibility of `work pools` tutorial — https://github.com/PrefectHQ/prefect/pull/12747
+- Fix `typing` import and typos in Tasks tutorial — https://github.com/PrefectHQ/prefect/pull/12746
```
Comment (serinamarie) [inline RELEASE-NOTES.md:72]:
```suggestion
- Fix `typing` import and typos in tasks tutorial — https://github.com/PrefectHQ/prefect/pull/12746
```

File: RELEASE-NOTES.md [hunk 16]
```diff
@@ -1,5 +1,96 @@
...
+- Remove duplicative resilient — https://github.com/PrefectHQ/prefect/pull/12723
+- Fix typo in Quickstart — https://github.com/PrefectHQ/prefect/pull/12729
+- Simplify-api-connection — https://github.com/PrefectHQ/prefect/pull/12725
+- Add `.serve`, `.deploy`, and composite trigger examples to deployment triggers docs — https://github.com/PrefectHQ/prefect/pull/12743
+- Update automations images — https://github.com/PrefectHQ/prefect/pull/12752
+- Simplify tutorial — https://github.com/PrefectHQ/prefect/pull/12765
+- Remove disclaimer for Python 3.12 experimental support — https://github.com/PrefectHQ/prefect/pull/12771
+- Clarify deployment trigger examples — https://github.com/PrefectHQ/prefect/pull/12782
+- Remove Prefect-managed integration libraries to be archived from the integrations catalog — https://github.com/PrefectHQ/prefect/pull/12781
+- Fix broken link to Push Work Pool Guide — https://github.com/PrefectHQ/prefect/pull/12748
```
Comment (serinamarie) [inline RELEASE-NOTES.md:70]:
```suggestion
- Fix broken link to push work pool guide — https://github.com/PrefectHQ/prefect/pull/12748
```

File: RELEASE-NOTES.md [hunk 17]
```diff
@@ -1,5 +1,96 @@
...
+- Fix minor restructure to improve legibility of `work pools` tutorial — https://github.com/PrefectHQ/prefect/pull/12747
```
Comment (serinamarie) [inline RELEASE-NOTES.md:71]:
```suggestion
- Fix minor restructure to improve legibility of work pools tutorial — https://github.com/PrefectHQ/prefect/pull/12747
```

File: RELEASE-NOTES.md [hunk 18]
```diff
@@ -1,5 +1,96 @@
...
+- Rename `prefect-cloud.*` events and labels to `prefect.*` — https://github.com/PrefectHQ/prefect/pull/12755
+- Add ability to emit events to an ephemeral Prefect server — https://github.com/PrefectHQ/prefect/pull/12762
+- Disable `events` and `automations` API routes when experimental events setting is not enabled — https://github.com/PrefectHQ/prefect/pull/12777
+- Add compatibility tests for client and server triggers and actions — https://github.com/PrefectHQ/prefect/pull/12778
+- Disable the automations integration flows for Prefect Cloud — https://github.com/PrefectHQ/prefect/pull/12784
+- Implement the work pool and work queue actions — https://github.com/PrefectHQ/prefect/pull/12735
+- Add helper functions for creating an events client or subscriber — https://github.com/PrefectHQ/prefect/pull/12759
+
+### Documentation
+- Update example to `prefect.yaml` in  work-pools concepts page — https://github.com/PrefectHQ/prefect/pull/12695
```
Comment (serinamarie) [inline RELEASE-NOTES.md:57]:
```suggestion
- Update `prefect.yaml` example in work pools concepts page — https://github.com/PrefectHQ/prefect/pull/12695
```

File: RELEASE-NOTES.md [hunk 19]
```diff
@@ -1,5 +1,96 @@
...
+- Fix stray link — https://github.com/PrefectHQ/prefect/pull/12721
+- Add hyphen — https://github.com/PrefectHQ/prefect/pull/12724
+- Fix typo — https://github.com/PrefectHQ/prefect/pull/12722
```
Comment (serinamarie) [inline RELEASE-NOTES.md:60]: these may not be worth including, esp if we don't feel like being descriptive

File: RELEASE-NOTES.md [hunk 20]
```diff
@@ -1,5 +1,96 @@
...
+- Remove duplicative resilient — https://github.com/PrefectHQ/prefect/pull/12723
```
Comment (serinamarie) [inline RELEASE-NOTES.md:61]: same with this one

File: RELEASE-NOTES.md [hunk 21]
```diff
@@ -1,5 +1,96 @@
...
+- Fix typo in Quickstart — https://github.com/PrefectHQ/prefect/pull/12729
+- Simplify-api-connection — https://github.com/PrefectHQ/prefect/pull/12725
```
Comment (serinamarie) [inline RELEASE-NOTES.md:63]:
```suggestion
- Simplify quickstart — https://github.com/PrefectHQ/prefect/pull/12725
```

File: RELEASE-NOTES.md [hunk 22]
```diff
@@ -1,5 +1,96 @@
...
+- Allow for creating automations via deployments when experimental events is on — https://github.com/PrefectHQ/prefect/pull/12701
+- Add ability to stream out events via websocket — https://github.com/PrefectHQ/prefect/pull/12744
+- Implementing the `pause-automation` and `resume-automation` actions — https://github.com/PrefectHQ/prefect/pull/12738
+- Add an Automations CLI — https://github.com/PrefectHQ/prefect/pull/12754
+- Rename `prefect-cloud.*` events and labels to `prefect.*` — https://github.com/PrefectHQ/prefect/pull/12755
+- Add ability to emit events to an ephemeral Prefect server — https://github.com/PrefectHQ/prefect/pull/12762
+- Disable `events` and `automations` API routes when experimental events setting is not enabled — https://github.com/PrefectHQ/prefect/pull/12777
+- Add compatibility tests for client and server triggers and actions — https://github.com/PrefectHQ/prefect/pull/12778
+- Disable the automations integration flows for Prefect Cloud — https://github.com/PrefectHQ/prefect/pull/12784
+- Implement the work pool and work queue actions — https://github.com/PrefectHQ/prefect/pull/12735
```
Comment (billpalombi) [inline RELEASE-NOTES.md:53]:
```suggestion
- Add pause and resume the work pool and work queue actions — https://github.com/PrefectHQ/prefect/pull/12735
```

File: RELEASE-NOTES.md (hunk 19 above)
Comment (billpalombi) [reply RELEASE-NOTES.md]: +1

File: RELEASE-NOTES.md [hunk 23]
```diff
@@ -1,5 +1,90 @@
...
+- Add automations CLI — https://github.com/PrefectHQ/prefect/pull/12754
+- Rename `prefect-cloud.*` events and labels to `prefect.*` — https://github.com/PrefectHQ/prefect/pull/12755
+- Add ability to emit events to an ephemeral Prefect server — https://github.com/PrefectHQ/prefect/pull/12762
+- Disable `events` and `automations` API routes when experimental events setting is not enabled — https://github.com/PrefectHQ/prefect/pull/12777
+- Add compatibility tests for client and server triggers and actions — https://github.com/PrefectHQ/prefect/pull/12778
+- Disable the automations integration flows for Prefect Cloud — https://github.com/PrefectHQ/prefect/pull/12784
+- Implement the work pool and work queue actions — https://github.com/PrefectHQ/prefect/pull/12735
+- Add helper functions for creating an events client or subscriber — https://github.com/PrefectHQ/prefect/pull/12759
+- Add default posture to `EventTrigger` schema — https://github.com/PrefectHQ/prefect/pull/12764
+ - Fix writing events for SQLite + SQLAlchemy<2 — https://github.com/PrefectHQ/prefect/pull/12679
```
Comment (billpalombi) [inline RELEASE-NOTES.md:50]:
```suggestion
- Fix writing events for SQLite + SQLAlchemy<2 — https://github.com/PrefectHQ/prefect/pull/12679
```

File: RELEASE-NOTES.md [hunk 24]
```diff
@@ -1,5 +1,90 @@
...
+
+### Documentation
+- Update `prefect.yaml` example in work pools concepts page — https://github.com/PrefectHQ/prefect/pull/12695
+- Fix typo in Quickstart — https://github.com/PrefectHQ/prefect/pull/12729
```
Comment (billpalombi) [inline RELEASE-NOTES.md:54]:
```suggestion
- Fix typo in quickstart — https://github.com/PrefectHQ/prefect/pull/12729
```

File: RELEASE-NOTES.md [hunk 25]
```diff
@@ -1,5 +1,83 @@
...
+- Simplify tutorial — https://github.com/PrefectHQ/prefect/pull/12765
+- Remove disclaimer for Python 3.12 experimental support — https://github.com/PrefectHQ/prefect/pull/12771
+- Clarify deployment trigger examples — https://github.com/PrefectHQ/prefect/pull/12782
+- Remove Prefect-managed integration libraries to be archived from the integrations catalog — https://github.com/PrefectHQ/prefect/pull/12781
+- Fix broken link to push work pool guide — https://github.com/PrefectHQ/prefect/pull/12748
+- Fix minor restructure to improve legibility of work pools tutorial — https://github.com/PrefectHQ/prefect/pull/12747
+- Fix `typing` import and typos in tasks tutorial — https://github.com/PrefectHQ/prefect/pull/12746
+- Simplify installation — https://github.com/PrefectHQ/prefect/pull/12772
+- Fix import syntax in `variables.Variable` example — https://github.com/PrefectHQ/prefect/pull/12727
+- Fix typo in How-to Guide document — https://github.com/PrefectHQ/prefect/pull/12761
```
Comment (billpalombi) [inline RELEASE-NOTES.md:67]:
```suggestion
- Fix typo in How-to Guide — https://github.com/PrefectHQ/prefect/pull/12761
```

**Review by billpalombi — APPROVED** [review]
I made a few other suggestions but this is good enough to keep moving

**Pull request merged**